packages/bind-mount/index.js

```diff
@@ -0,0 +1,35 @@
+try {
+  const addon = require('bindings')('addon');
+
+  module.exports.mount = (source, target) => {
+    return new Promise((resolve, reject) => {
+      addon.Mount(source, target, (err) => {
+        if (err) {
+          reject(err);
+        } else {
+          resolve();
+        }
+      });
+    });
+  };
+
+  module.exports.unmount = (target) => {
```

Comment:
Not that I really mind, but would this be clearer as `umount()` to match the C lib function and the command line utility names?

Reply:
That makes sense, yeah. Will change.